src/main/java/org/folio/linked/data/controller/ResourceController.java

```diff
@@ -1,24 +1,23 @@
 package org.folio.linked.data.controller;
 
+import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
 import org.folio.linked.data.domain.dto.ResourceDto;
 import org.folio.linked.data.domain.dto.ResourceMarcViewDto;
 import org.folio.linked.data.domain.dto.ResourceShortInfoPage;
 import org.folio.linked.data.rest.resource.ResourceApi;
 import org.folio.linked.data.service.ResourceService;
 import org.springframework.http.ResponseEntity;
-import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.RestController;
 
-@Validated
 @RestController
 @RequiredArgsConstructor
 public class ResourceController implements ResourceApi {
 
   private final ResourceService resourceService;
 
   @Override
-  public ResponseEntity<ResourceDto> createResource(String okapiTenant, ResourceDto resourceDto) {
+  public ResponseEntity<ResourceDto> createResource(String okapiTenant, @Valid ResourceDto resourceDto) {
```

Comment:
Hi @PBobylev,

Although the addition of the @Valid annotation is an improvement, I don't think it will prevent the issue of the missing field 245 in the generated MARC.

If all the titles in the incoming Work DTO object are of type ParallelTitleField or VariantTitleField, the validation will pass (since the title collection is not empty). However, MARC 245 will still not be generated because it is derived from `BasicTitle` objects.

To ensure that MARC 245 is generated correctly, we must validate that the list contains exactly one BasicTitle object.

Reply:
I see, will improve the validation and include such check

Reply:
done